### PR TITLE
feat: docker settings page

### PR DESCRIPTION
**Pull request by tarnas14**
Description
---

- [x] prepare docker settings page
- [x] get docker images list from backend
- [x] prepare UI for docker image pull
- [x] update docker images when opening settings
- [ ] connect backend commands for pulling ima ges to settings - next PR after #311 
- [ ] listen to docker pull events and update store - next PR after #311

Motivation and Context
---

#28 

How Has This Been Tested?
---

dockersettings

loading state:
loading_state_docker_settings

### CODE REVIEW

**Comment by tarnas14** [comment]
I'm wondering if it would be better to put 'docker' in a separate slice ? not sure :\

**Comment by tarnas14** [comment]
> I'm wondering if it would be better to put 'docker' in a separate slice ? not sure :\

moved to a separate slice, as discussed on slack

File: applications/launchpad/gui-react/src/containers/SettingsContainer/DockerSettings/DockerImagesList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  const dockerImages = useAppSelector(selectDockerImages)
+  const dockerImagesLoading = useAppSelector(selectDockerImagesLoading)
+
+  return (
+    <DockerList>
+      {dockerImagesLoading && <LoadingOverlay />}
+      {dockerImages.map(dockerImage => (
+        <DockerRow key={dockerImage.dockerImage}>
+          <Text style={{ flexBasis: '40%' }}>{dockerImage.displayName}</Text>
```
Comment (tomaszantas) [inline applications/launchpad/gui-react/src/containers/SettingsContainer/DockerSettings/DockerImagesList/index.tsx:36]: Here is the issue on macOS:

image

...but if we change the `flexBasis` to `38%`, it looks good:

image

I can handle this later and check also how it looks on the Windows.

**Review by tomaszantas — APPROVED** [review]
Found one thing, but we can create an issue for it and handle it later.

**Pull request merged**